### ISSUE TITLE
nginx: Invalid `SystemCallFilter`

### ISSUE THREAD

**Issue by erikarvstedt** [NixOS/nixpkgs#148009]
https://github.com/NixOS/nixpkgs/pull/146268 introduced the invalid `SystemCallFilter` item `@mincore` to nginx.
`mincore` is a system call, not a systemd syscall group, so the correct syntax would be `mincore`.

What's the reason for prohibiting `mincore`? nginx is the only NixOS service where this syscall is explicitly excluded.

#### Reproduce

```bash
nixpkgsUnstable=https://github.com/NixOS/nixpkgs/archive/8a308775674e178495767df90c419425474582a1.tar.gz
extra-container shell --nixpkgs-path "fetchTarball $nixpkgsUnstable" -E '
{
  containers.tmp.config.services.nginx.enable = true;
}
' --run c systemd-analyze verify -

# => /etc/systemd/system/nginx.service:54: Unknown system call group, ignoring: @mincore
```

cc @ajs124, @Izorkin, @dasJ

**Comment by ajs124** [comment]
hm. It would probably be from some library. Maybe openssl? But now I can't reproduce it either, weird.

Maybe we can just revert c408cd921f8681df1eb813a8692ed740dacfe2d4

`nixosTests.nginx-variants.tengine` seems to fail on master anyways.

**Comment by erikarvstedt** [comment]
Sorry, `mincore` is defined as a forbidden syscall, not as an allowed one as I had initially assumed.
I've edited the issue text.

**Comment by erikarvstedt** [comment]
@ajs124, what's the reason for prohibiting `mincore`? nginx is the only NixOS service where this syscall is explicitly excluded.

**Comment by Mic92** [comment]
Here some reasoning why systemd dropped this syscall from its default set: https://github.com/systemd/systemd/blob/285a6908eb08f32b51e5ca9eae8a9317179d33c4/NEWS#L4476

**Comment by Izorkin** [comment]
Filter `mincore` can block the detection of the presence of the kTLS module in the kernel?

**Comment by erikarvstedt** [comment]
Fixed in #148695.
